notebooks/JuMP-Sudoku.ipynb

```diff
@@ -317,29 +305,25 @@
   {
    "cell_type": "code",
    "execution_count": 7,
-   "metadata": {
-    "collapsed": false
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
     "# solve problem\n",
-    "JuMP.optimize(sudoku)"
+    "JuMP.optimize!(sudoku)"
```

Comment:
No need for the `JuMP.` prefix, it is now exported

Reply:
Updated :smiley: